### PR TITLE
Remove word embeddings

### PR DESCRIPTION
**Pull request by rudigiesler**
## Purpose
Word embedding has been creating issues with developer setup, and packaging.

## Solution
We want to remove the word embedding based search feature from ContentRepo.

#### Checklist
- [x] Added or updated unit tests
- [x] Added to release notes
- [x] Updated readme/documentation (if necessary)

### CODE REVIEW

**Review by jerith — DISMISSED** [review]
🎉 (Assuming we can actually do it.)

**Comment by KaitCrawford** [comment]
@rudigiesler this feature is no longer in use by the WHO instance and it can be removed from the application

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!--
 ## Unreleased
-
+- Removed word embeddings search (`s` query parameter in API)
```
Comment (jerith) [inline CHANGELOG.md:10]: Should this have a `### Removed` header?

**Review by jerith — APPROVED** [review]
🎉

**Review by jerith — APPROVED** [review]

**Pull request merged**